dev/src/bulk-writer.ts

```diff
@@ -65,6 +71,12 @@ export const DEFAULT_INITIAL_OPS_PER_SECOND_LIMIT = 500;
  */
 export const DEFAULT_MAXIMUM_OPS_PER_SECOND_LIMIT = 10000;
 
+/*!
+ * The default jitter to apply. For example, a factor of 0.3 means a 30% jitter
+ * is applied.
+ */
```

Comment:
This should mention that this relates to backoff. It needs some context in this module.

Reply:
Done.